assets/cms/components/IconButton.vue

```diff
@@ -0,0 +1,68 @@
+<template functional>
+    <span class="button-container">
+        <button
+            @click='listeners.click'
+            :class="[{ 'btn btn-light' : props.format === 'floating',
+                    'btn btn-outline-secondary' : props.format === 'outline',
+                    'btn btn-outline-primary' : props.format === 'outline-primary',
+                }]"
+            :disabled="props.disabled"
+            >
+            <span class="label" v-if="props.labelPosition === 'left'"
+                :class="{disabled : props.disabled}">
```

Comment:
Our code style guidelines would have this be `{ disabled: props.disabled }` please update to match and try to follow our code style guidelines everywhere. We'll need to get automated lint working for these things too.

Reply:
Also I don't think this span is serving any purpose